packages/google_maps_flutter/google_maps_flutter_ios/example/ios12/ios/RunnerTests/GoogleMapPolylineController_Test.h

```diff
@@ -0,0 +1,13 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#import "GoogleMapPolylineController.h"
+
+@interface FLTGoogleMapPolylineController (Test)
```

Comment:
Needs a declaration comment, per style guide. Something like "Private APIs exposed for unit testing."